### DISCUSSION TITLE
Getting " Gmail_API: Recipient address required"

### DISCUSSION THREAD

**Discussion by NickHodges** [emailjs-com/help#52] [Q&A]
Okay, I'm having trouble sending an email with GMail.  All my current attempts get:

`Gmail_API: Recipient address required`

The code I'm using to call Emailjs is:

```
 await emailjs.send(emailjsServiceId, emailjsTemplateId, {
      reply_to: 'nickhodges@gmail.com',
      email: emailAddress,
      user_email: emailAddress,
    });
```

It isn't clear to me what the difference between {{ email }} and {{ user_email }} is, and the docs don't tell me.  

The definition of the data structure to be passed in the third parameter is not clear to me either, and the docs don't seem to tell me. 

Any help?

**Comment by xr0master** [comment]
I think the error message is pretty clear, right? EmailJS doesn't create dynamic variables, but the customer creates them.
I suggest you carefully read the [tutorial](https://www.emailjs.com/docs/tutorial/overview/) and the [dynamic variables docs](https://www.emailjs.com/docs/user-guide/dynamic-variables-templates/).

**Comment by NickHodges** [comment]
Okay -- I guess I misunderstood. Thank you.
